wmn-data.json

```diff
@@ -3019,12 +2959,12 @@
         "cat" : "social"
        },
        {
-        "name" : "love_ru",
-        "uri_check" : "https://love.ru/{account}",
+        "name" : "waytohey",

```

Comment:
Please move this to the W section of the JSON file instead of keeping it here in the L-M alphabetized section.